tsk/pool/pool_open.cpp

```diff
@@ -3,16 +3,19 @@
  *
  * Brian Carrier [carrier <at> sleuthkit [dot] org]
  * Copyright (c) 2018-2019 BlackBag Technologies.  All Rights reserved
+ * Copyright (c) 2022 Joachim Metz <joachim.metz@gmail.com>
  *
  * This software is distributed under the Common Public License 1.0
  */
-#include "../fs/tsk_apfs.hpp"
+#include "tsk/tsk_config.h"
```

Comment:
This breaks the Visual Studio build since the file is not found.

Reply:
what is not found? can't this be addressed with an additional include path in the VS configuration?

Reply:
tsk_config.h does not exist.

Reply:
ah so it is not generated for VS, any conditional imports I should use? what is the recommended way of dealing with this in this codebase?

Reply:
You could try importing tsk_base_i.h instead - it has a check on HAVE_CONFIG_H before trying to include it.

Reply:
I'll have a look at that (tomorrow local time), the VS configuration has to define HAVE_LIBBFIO and HAVE_LIBVSLVM and ideally LIBVSLVM_HAVE_BFIO as well

Reply:
Made changes to headers